README.md

```diff
@@ -252,22 +270,36 @@ To run datalake from command line you have to run the below gradle command with
 To run freeipa management service from command line you have to run the below gradle command with the following list of JVM parameters:
 
 ````
-./gradlew :freeipa:bootRun -PjvmArgs="-Dfreeipa.db.addr=localhost \
--Dserver.port=8090 \
+./gradlew :freeipa:bootRun -PjvmArgs="\
+-Denvironment.db.port.5432.tcp.addr=localhost \
+-Dserver.port=8087 \
+-Denvironment.cloudbreak.url=http://localhost:8080
 -Dvault.addr=localhost \
 -Dvault.root.token=<TOKEN_FROM_PROFILE_FILE> \
 -Dspring.config.location=$(pwd)/freeipa/src/main/resources/application.yml,$(pwd)/freeipa/build/resources/main/application.properties"
 ````
 
+### Environment
+To run environment service from command line you have to run the below gradle command with the following list of JVM parameters:
+
+````
+./gradlew :environment:bootRun -PjvmArgs="\
+-Denvironment.db.addr=localhost \
+-Dserver.port=8090 \
```

Comment:
Shouldn't it match the port in [CBD](https://github.com/hortonworks/cloudbreak-deployer/blob/f030ba942d5a496c7bbafe6e801eac38924cd0f7/include/deployer.bash#L407) (ie. 8088)?